web/themes/custom/novel/novel.theme

```diff
@@ -276,3 +277,43 @@ function novel_theme_suggestions_select_alter(array &$suggestions, array $variab
     $suggestions[] = 'select__single';
   }
 }
+
+/**
+ * Preprocesses rows in a Drupal view to handle the display of event instances.
+ *
+ * This function iterates over each row in the view, checking for the presence
+ * of an EventInstance object.
+ * When an EventInstance is found, it checks if it belongs to the same
+ * event series as the previous row.
+ * If so, it marks the current row as 'stacked' and adjusts its view
+ * mode accordingly.
+ * This is useful for grouping events from the same series in the view,
+ * enhancing visual continuity and clarity.
```

Comment:
This smells like a CoPilot commenting :smile:. It is not really helpful except for the last line. It tries to get close to why we are doing this.